tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/llk_math_eltwise_unary_sfpu_macros.h

```diff
@@ -0,0 +1,176 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
```

Comment:
Dumb suggestion: is there any difference between this file and the other one? If not can we make it a symlink instead of a duplicate file?